### PR TITLE
add solution to kata 8

### PR DESCRIPTION
**Pull request by milosh1982**

### CODE REVIEW

File: models/article.model.js [hunk 1]
```diff
@@ -65,3 +65,28 @@ exports.selectComments = (id) => {
       return rows;
     });
 };
+
+exports.selectPatchVotesArticle = (id, patch) => {
+  if (!patch || !patch.inc_votes) {
+    return Promise.reject({ status: 400, msg: "Invalid Request" });
+  }
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [id])
```
Comment (yalicee) [inline models/article.model.js]: Is it necessary to check the id exists with a separate query?

File: models/article.model.js [hunk 2]
```diff
@@ -65,3 +65,28 @@ exports.selectComments = (id) => {
...
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Not found" });
+      }
+      return rows[0];
+    })
+    .then((articleRow) => {
+      const newVotesValue = articleRow.votes + patch.inc_votes;
+      return db
+        .query(
+          `UPDATE articles SET votes = ${newVotesValue} WHERE article_id = $1 RETURNING *;`,
```
Comment (yalicee) [inline models/article.model.js]: Refactor to prevent sql injection

**Review by yalicee — APPROVED** [review]
Minor changes before you merge

File: models/article.model.js (hunk 1 above)
Comment (milosh1982) [reply models/article.model.js]: the main reason for that query is to grab votes from articles and then later to add to that new vote. and on top of that I could check if id exist, so don't understand what is wrong with this approach. I need to grab votes from articles somehow.

Comment (milosh1982) [reply models/article.model.js]: ok. changed so there is only one db query. thank you for feedback.

**Pull request merged**